sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -78,10 +79,15 @@ import {
   DecryptResult,
   EncryptOptions,
   EncryptResult,
+  KeySignatureAlgorithm,
   KeyWrapAlgorithm,
+  SignOptions,
   SignResult,
   UnwrapResult,
+  UnwrapKeyOptions,
+  VerifyOptions,
   VerifyResult,
+  WrapKeyOptions,
```

Comment:
@jonathandturner I assumed that this PR is only docs related, but it looks like we are also updating what gets exported from say key vault keys package, like we are doing here.

How do these missing exports affect the package we just shipped?

Reply:
These would impact some of the things the user is able to type of their side. Any non-exported interface that they need (like these *Options interfaces) would not be import-able on their side.